src/translib/transformer/xfmr_intf.go

```diff
@@ -1932,6 +1932,12 @@ var YangToDb_intf_eth_port_config_xfmr SubTreeXfmrYangToDb = func(inParams XfmrP
                     errStr := "Interface already part of a PortChannel"
                     return nil, tlerr.InvalidArgsError{Format: errStr}
                 }
+                /* Restrict configuring member-port if iface configured as member-port of any vlan */
+                err = validateIntfAssociatedWithVlan(inParams.d, &ifName)
+                if err != nil {
+                    errStr := "PortChannel config not permitted on Vlan member-port"
```

Comment:
Better to address here as well, it would be easier for the customer to fix the issue.